plugins/org.jboss.tools.openshift.ui/plugin.xml

```diff
@@ -978,6 +978,34 @@
            commandId="org.eclipse.ui.edit.delete"
            style="push" />
   </menuContribution>
+  <menuContribution
+        allPopups="false"
+        locationURI="popup:org.jboss.tools.openshift.ui.properties.tab.StorageTab">
+     <command
+           commandId="org.jboss.tools.openshift.ui.command.resource.refresh"
+           icon="icons/refresh.gif"
+           id="org.jboss.tools.openshift.ui.command.resource.refresh"
+           style="push"
+           tooltip="Refresh the resource">
```

Comment:
Agreed that having a simple "Refresh" command is self-describing, for all resource objects. The user when-right clicking knows what he's refreshing.